### PR TITLE
Add a readme

### PR DESCRIPTION
**Pull request by arjun024**
Signed-off-by: Arjun Sreedharan <asreedharan@vmware.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,78 @@
...
+To package this buildpack for consumption:
+
+```
+$ ./scripts/package.sh --version <version-number>
+```
+
+This will create a `buildpackage.cnb` file under the `build` directory which you
+can use to build your app as follows:
+```
+pack build <app-name> -p <path-to-app> -b build/buildpackage.cnb \
```
Comment (thitch97) [inline README.md:60]: Would this work as-is? Wouldn't they need `node-engine` since it begins the order grouping?

Comment (thitch97) [inline README.md:60]: Would this work as-is? Though the `yarn` buildpack itself wouldn't require `node-engine`, wouldn't it still need to be present for buildpacks that would require yarn?

Comment (arjun024) [reply README.md]: That does make sense.
Also, given that yarn is unusable without node, does it make sense to change the API of this buidpack to require node?
Currently, this buildpack's output is as follows:
```
cnb@715a16255014:/workspace$ yarn --help
Yarn requires Node.js 4.0 or higher to be installed.
```

Comment (ryanmoran) [reply README.md]: I don't think that it makes sense to require `node` here. The buildpack does not use `yarn` and therefore does not need `node` to perform its role. It isn't until we want to run a `yarn` command, like in `yarn-install` that you really need to required `node`. Any requirement of `node` before that point would be presumptuous and limit the ways in which the buildpack could be used.

**Review by thitch97 — APPROVED** [review]

**Pull request merged**